### PR TITLE
dev, v2.1, v3.0: fix broken links

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 

Fix broken links

### Which version does your change affect? 

dev, v2.1, v3.0

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

@lilin90 PTAL

### CODE REVIEW

**Comment by CaitinChen** [comment]
@lilin90 PTAL

File: dev/faq/tidb.md [hunk 1]
```diff
@@ -524,12 +524,12 @@ TiDB 在执行 SQL 时，预估出来每个 operator 处理了超过 10000 条
 
 #### 3.3.10 在 TiDB 中如何控制或改变 SQL 提交的执行优先级？
 
-TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sq(/reference/configuration/tidb-server/server-command-option.md#force-priority)或[单个语句](/reference/sql/statements/dml.md)的优先级。优先级包括：
+TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sq(/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
```
Comment (yikeke) [inline dev/faq/tidb.md]:
```suggestion
TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sql/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
```

File: v2.1/faq/tidb.md [hunk 2]
```diff
@@ -524,12 +524,12 @@ TiDB 在执行 SQL 时，预估出来每个 operator 处理了超过 10000 条
 
 #### 3.3.10 在 TiDB 中如何控制或改变 SQL 提交的执行优先级？
 
-TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sq(/reference/configuration/tidb-server/server-command-option.md#force-priority)或[单个语句](/reference/sql/statements/dml.md)的优先级。优先级包括：
+TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sq(/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
```
Comment (yikeke) [inline v2.1/faq/tidb.md]:
```suggestion
TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sql/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
```

File: v3.0/faq/tidb.md [hunk 3]
```diff
@@ -525,12 +525,12 @@ TiDB 在执行 SQL 时，预估出来每个 operator 处理了超过 10000 条
 
 #### 3.3.10 在 TiDB 中如何控制或改变 SQL 提交的执行优先级？
 
-TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sq(/reference/configuration/tidb-server/server-command-option.md#force-priority)或[单个语句](/reference/sql/statements/dml.md)的优先级。优先级包括：
+TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sq(/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
```
Comment (yikeke) [inline v3.0/faq/tidb.md]:
```suggestion
TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sql/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
```

File: dev/faq/tidb.md [hunk 4]
```diff
@@ -524,12 +524,12 @@ TiDB 在执行 SQL 时，预估出来每个 operator 处理了超过 10000 条
...
-TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sq(/reference/configuration/tidb-server/server-command-option.md#force-priority)或[单个语句](/reference/sql/statements/dml.md)的优先级。优先级包括：
+TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sql/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
 
 - HIGH_PRIORITY：该语句为高优先级语句，TiDB 在执行阶段会优先处理这条语句
 - LOW_PRIORITY：该语句为低优先级语句，TiDB 在执行阶段会降低这条语句的优先级
 
-以上两种参数可以结合 TiDB 的 DML 语言进行使用，具体使用方式可以参考[官方文档](/reference/sql/statements/dml.md)，使用方法举例如下：
+以上两种参数可以结合 TiDB 的 DML 语言进行使用，使用方法举例如下：
 
 1）通过在数据库中写 SQL 的方式来调整优先级：
```
Comment (lilin90) [inline dev/faq/tidb.md]: Let's update this list to the strict format.

```suggestion
1. 通过在数据库中写 SQL 的方式来调整优先级：
```

File: dev/faq/tidb.md (hunk 1 above)
Comment (lilin90) [reply dev/faq/tidb.md]: @yikeke 😂

File: dev/faq/tidb.md [hunk 5]
```diff
@@ -524,24 +524,24 @@ TiDB 在执行 SQL 时，预估出来每个 operator 处理了超过 10000 条
 
 #### 3.3.10 在 TiDB 中如何控制或改变 SQL 提交的执行优先级？
 
-TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sq(/reference/configuration/tidb-server/server-command-option.md#force-priority)或[单个语句](/reference/sql/statements/dml.md)的优先级。优先级包括：
+TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sql/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
```
Comment (lilin90) [inline dev/faq/tidb.md]:
```suggestion
TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
```

File: v2.1/faq/tidb.md [hunk 6]
```diff
@@ -524,24 +524,24 @@ TiDB 在执行 SQL 时，预估出来每个 operator 处理了超过 10000 条
 
 #### 3.3.10 在 TiDB 中如何控制或改变 SQL 提交的执行优先级？
 
-TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sq(/reference/configuration/tidb-server/server-command-option.md#force-priority)或[单个语句](/reference/sql/statements/dml.md)的优先级。优先级包括：
+TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sql/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
```
Comment (lilin90) [inline v2.1/faq/tidb.md]:
```suggestion
TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
```

File: v3.0/faq/tidb.md [hunk 7]
```diff
@@ -525,24 +525,24 @@ TiDB 在执行 SQL 时，预估出来每个 operator 处理了超过 10000 条
 
 #### 3.3.10 在 TiDB 中如何控制或改变 SQL 提交的执行优先级？
 
-TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sq(/reference/configuration/tidb-server/server-command-option.md#force-priority)或[单个语句](/reference/sql/statements/dml.md)的优先级。优先级包括：
+TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](sql/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
```
Comment (lilin90) [inline v3.0/faq/tidb.md]:
```suggestion
TiDB 支持改变 [per-session](/reference/configuration/tidb-server/tidb-specific-variables#tidb_force_priority)、[全局](/reference/configuration/tidb-server/server-command-option.md#force-priority)或单个语句的优先级。优先级包括：
```

**Comment by CaitinChen** [comment]
@lilin90 PTAL gain. Thanks.

**Pull request merged**